.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "datalake/backend/processing/check_metadata_content/stac-spec"]
```

Comment:
Deployment fails on 

```
COPY failed: stat /var/lib/docker/tmp/docker-builder008079860/datalake/backend/processing/check_metadata_content/stac-spec/collection-spec/json-schema/collection.json: no such file or directory
[100%] fail: docker build --tag cdkasset-105dc3b20ec783a2e00ff6b5d249f6a7a9d65e41212c4518c36ceb5b14a5f419 --file datalake/backend/processing/check_metadata_content/Dockerfile . exited with error code 1: COPY failed: stat /var/lib/docker/tmp/docker-builder008079860/datalake/backend/processing/check_metadata_content/stac-spec/collection-spec/json-schema/collection.json: no such file or directory
```

We need to add something like below to our `reset-dev-env.bash` script

```
git submodule init 
git submodule update
```